### PR TITLE
WIP: about page

### LINKED ISSUE #36: About page
Create about page with details, creators and links etc.

Design deppends on #28 and #29.

On mobile, the button to get there might be in a drawer.

* [ ] Web
* [ ] Mobile

### PR DESCRIPTION
**Pull request by tenhobi**
Fixes #36.

Changes proposed in this pull request:

*
*
*

### CODE REVIEW

File: flutter/lib/src/about_page.dart [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  Widget build(BuildContext context) {
+    return new Scaffold(
+      backgroundColor: Theme.of(context).primaryColor,
+      appBar: new AppBar(
+        title: const Text('About flashcards'),
+        elevation: 0.0,
+      ),
+      body: new ListView (
+        padding: const EdgeInsets.all(16.0),
+        children: <Widget>[
```
[the remark this answers is not in the document]
Comment (michaldrabina) [reply flutter/lib/src/about_page.dart]: Problem is that this markdown doesn't properly support breaks between paragraphs so this is my workaround.

File: flutter/lib/main.dart [hunk 2]
```diff
@@ -24,7 +24,7 @@ class MyApp extends StatelessWidget {
       child: MaterialApp(
         title: 'Flashcards',
         theme: ThemeData(
-          primarySwatch: Colors.blue,
+          primaryColor: Colors.blue[800]
```
[the remark this answers is not in the document]
Comment (michaldrabina) [reply flutter/lib/main.dart]: I wanted the colour to be consistent across the app and the old one was pretty bad for the use on the page. The change isn't that big I'd say and it helps a lot.

File: flutter/lib/src/about_page.dart (hunk 1 above)
[the remark this answers is not in the document]
Comment (tenhobi) [reply flutter/lib/src/about_page.dart]: Really? 😮 Hmm, I will try to take a look at it later.

[the remark this answers is not in the document]
Comment (michaldrabina) [reply flutter/lib/src/about_page.dart]: https://github.com/flutter/flutter_markdown/issues/40

File: flutter/lib/main.dart (hunk 2 above)
[the remark this answers is not in the document]
Comment (tenhobi) [reply flutter/lib/main.dart]: OK then.

File: flutter/lib/src/about_page.dart (hunk 1 above)
[the remark this answers is not in the document]
Comment (tenhobi) [reply flutter/lib/src/about_page.dart]: Well, that issue is related to something else, I believe. You can set up space between paragraphs like this:

image

[the remark this answers is not in the document]
Comment (michaldrabina) [reply flutter/lib/src/about_page.dart]: I must've overlooked this. My bad. This still means that there is no way to have warying size spaces between paragraphs by adding more like breaks. (Worth noting if we ever need that)

[the remark this answers is not in the document]
Comment (tenhobi) [reply flutter/lib/src/about_page.dart]: Yep. But we might do some "blocks" of text, if needed. Still better than this? 👍

**Comment by Kenny11CZ** [comment]
Version can't be displayed because PackageInfo.version returrns null instead of number specified in `pubspec.yaml`.
image

File: flutter/lib/main.dart (hunk 2 above)
[the remark this answers is not in the document]
Comment (tenhobi) [reply flutter/lib/main.dart]: Change the color only for that screen, please (probably some style, decoration or something like that property on Scaffold). This color look a bit weird as a main color.

**Comment by tenhobi** [comment]
I am finishing it now.

**Pull request merged**